src/test/java/DemoTest.java

```diff
@@ -1,7 +1,125 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
 
 public class DemoTest {
 
+  @Test
+  public void incorrectPassAndEmailTest() {
+    WebDriver driver = new ChromeDriver();
+    driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+    driver.get("https://deens-master.now.sh");
```

Comment:
use @BeforeMethod for preparing your WebDriver and open website because this part of code meet in each test method

Reply:
done